src/isomorphic/render.js

```diff
@@ -1,13 +1,21 @@
+/* @flow */
+
 import { buildComponentFromVNode } from './createComponent'
 
 let component;
 
+type Vnode = {
+  nodeName: string | ?void,
+  attributes: Array<any> | void,
+  children: string | Array<any> | void
+};
```

Comment:
Hey @matuzalemsteles , 

What do you think in create a separated folder, like flow-types and add a file, named as vnode.js, with the Vnode declarations?

Reply:
Sounds great, I liked it.

Reply:
o/